app/segments/drill/[id]/submission/input.js

```diff
@@ -28,13 +29,34 @@ import DrillTarget from "~/components/input/drillTarget";
 import NavigationRectangle from "~/components/input/navigationRectangle";
 import Loading from "~/components/loading";
 import { currentAuthContext } from "~/context/Auth";
+import { db } from "~/firebaseConfig";
 import Description from "./modals/description";
 
 /***************************************
  * Firebase Upload
  ***************************************/
 
-//TODO: Implement function to upload the outputData to the attempts collection
+//A function to upload the outputData to the "attempts" collection
+async function uploadAttempt(outputData) {
+  try {
+    //create new document
+    const newAttemptRef = doc(collection(db, "teams", "1", "attempts"));
+
+    //Newly created doc Id. Useful for finding upload data in testing.
+    console.log("New Attempt Ref ID: ", newAttemptRef.id);
+
+    //add id of new document into the data
+    const uploadData = { ...outputData, id: newAttemptRef.id };
```

Comment:
For some reason, this is creating a different formatting than the normal attempt. The data is wrapped in "outputData"
<img width="524" alt="Screenshot 2024-04-02 at 2 03 09 PM" src="https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/41023671/8bbee3a8-0083-42de-9a16-b3c3724e0b35">